<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>251</width>
      <height>203</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSlider" name="scale_slider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>threshold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>min</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>max size</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="thresholdBox"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="minBox"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="maxBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="log"/>
     </item>
    </layout>
   </widget>
   <widget class="ImageView" name="view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="s_analysis">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Analysis</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="discription">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="ellipseID">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>290</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Particle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_button">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>437</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="show">
       <property name="text">
        <string>Show source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scaling">
       <property name="text">
        <string>Set scaling</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="select_point">
       <property name="text">
        <string>( x , y )</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Scale:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="scale">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>[Î¼m]/pixel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="destruct">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>360</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Destruct</string>
    </property>
   </widget>
   <widget class="QPushButton" name="import_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>360</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QPushButton" name="apply_filter">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Filter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header>imageview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
